cmds/core/uptime/uptime.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Get the time the machine has been up
+//
+// Synopsis:
+//     uptime
+//
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+)
+
+// This function takes in the filepath to a file containing loadavgs,it then extracts and returns the three load averages
+func loadavg(filepath string) (loadaverage string, err error) {
+
+	loadavg := strings.Fields(filepath)
+	loadavgstring := "loadaverage :" + loadavg[0] + "," + loadavg[1] + "," + loadavg[2]
+	return loadavgstring, nil
```

Comment:
Why return this as a string?

Reply:
so we can have a type
type converter func([]byte) (string, error)
as a standard type. Then we get to use a loop. Then callers of this don't need to mess with formatting. We had the discussion this morning and went with a string.

Reply:
But we're not at that point yet, we're taking this one thing at a time.
Each function will return something suitable for concatenation to an output string.
Strings seem fine for this use.

Reply:
Done.I am returning formatted strings for now .I am open to changing the return types when we agree upon one.